### PR TITLE
edit lucybot for ios navigation

### PR DESCRIPTION
**Pull request by tzubeli**

### CODE REVIEW

File: ovp.LucyBot.yml [hunk 1]
```diff
@@ -557,3 +567,82 @@ workflowNavigation:
...
+  markdownURL: /assets/documentation/player-portal/ios/getting-started-ios.html
+  path: "/player/ios/getting"
+  discussionURL: null
+
+  - title: "Supported Devices for iOS"
+  markdownURL: /assets/documentation/player-portal/ios/supported-devices-ios.html
+  path: "/player/ios/supported-devices-ios.html"
+  discussionURL: null
+
+- title: "Player Events for iOS"
```
Comment (bobby-brennan) [inline ovp.LucyBot.yml]: the dash on this needs to be aligned with the above, ditto below

File: ovp.LucyBot.yml [hunk 2]
```diff
@@ -557,3 +567,82 @@ workflowNavigation:
...
+  markdownURL: /assets/documentation/player-portal/ios/player-events-ios.html
+  path: "/player/ios/player-events-ios.html"
+  discussionURL: null
```
Comment (bobby-brennan) [inline ovp.LucyBot.yml]: Assuming you don't want to show the discussion pages on all these - other wise you can unset this field.

Comment (tzubeli) [reply ovp.LucyBot.yml]: unset meaning leave it out?

Comment (bobby-brennan) [reply ovp.LucyBot.yml]: yup!

File: ovp.LucyBot.yml (hunk 1 above)
Comment (tzubeli) [reply ovp.LucyBot.yml]: fixed

File: workflows/ovp/create_and_manage_youtube_videos_as_native_kaltura_external_media_entry_/readme.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
+<!--METADATA
```
Comment (jessp01) [inline workflows/ovp/create_and_manage_youtube_videos_as_native_kaltura_external_media_entry_/readme.md]: @tzubeli - this pull seems to include an addition Zohar made in a different one and one that is unrelated to the matter at hand. Please remove this file from this branch.
